test/e2e_new/go.mod

```diff
@@ -0,0 +1,17 @@
+module sigs.k8s.io/cluster-api-provider-aws/test/e2e_new
```

Comment:
Is there a need currently for a separate go module, the main reason the existing e2e test has one is to avoid leaking the sonobuoy dependency (and any potential conflicts it brings) into the main go module.

Reply:
while adding more of the tests, I had to use CAPI@master. That's the only reason I could see and that's just temporary until a new release of capi is released and more of the test framework is available